refs/biblio.json

```diff
@@ -281,6 +281,90 @@
         "href": "https://standards.cta.tech/kwspub/published_docs/ANSI-CTA-708-E-Preview.pdf",
         "publisher": "CEA"
     },
+    "CEDDL": {
+        "authors": [
+            "Filip von Reiche",
+            "RMiles Fender",
+            "Murray Williams",
+            "Reza Jalili",
+            "Jay Myers",
+            "Jon Revill",
+            "Jason Walsh",
+            "Blane Sims",
+            "Jared Vestal",
+            "Eric Lunt",
+            "Paolo Margar",
+            "Patrick Wyatt",
+            "Thomas Brune",
+            "Sam Law",
+            "Simon Rumble",
+            "Anna Long",
+            "Eric Feinberg",
+            "Peter Loveday",
+            "Fares Aldik",
+            "Daniel Karpantschof",
+            "Mark Prince",
+            "Josh Goodwin",
+            "Joe Orlet",
+            "Mathieu Jondet",
+            "Willem Paling",
+            "Laura Holmes",
+            "Brian Kuhn",
+            "Justin Cutroni",
+            "Vishal Goenka",
+            "Lukas Bergstrom",
+            "Brian Hendrixson",
+            "Viswanath Srikanth",
+            "Aubrey Rupp",
+            "Hutch White",
+            "Casper Blicher Olsen",
+            "Amin Shawk",
+            "Andrew Thomas",
+            "Martijn van Berkum",
+            "Lee Isensee",
+            "Jonathan Weber",
+            "Gagan Kanwar",
+            "Emilie Laffray",
+            "William Hartley-Booth",
+            "Joseph Galarneau",
+            "Aurelie Pols",
+            "Anton Gething",
+            "Nicolas Malo",
+            "Roel Willems",
+            "Matt Stannard",
+            "Rod Jacka",
+            "Harry Hurst",
+            "Ian McCaig",
+            "Stephen Elliott",
+            "Keith Watkins",
+            "Jonathan Conway",
+            "Oliver Schiffers",
+            "Alexander Dean",
+            "Adam Ware",
+            "Toby Doig",
+            "Angus Glover Wilson",
+            "Ali Behnam",
+            "David Henderson",
+            "Frederic Abrioux",
+            "David Evans",
+            "Evan Klein"
+        ],
+        "href": "https://www.w3.org/2013/12/ceddl-201312",
+        "title": "Customer Experience Digital Data Layer",
+        "status": "BG-FINAL",
+        "publisher": "W3C",
+        "deliveredBy": [
+            "https://www.w3.org/community/custexpdata/"
+        ],
+        "versions": {
+            "1.0": {
+                "status": "BG-FINAL",
+                "rawDate": "2013-12-10",
+                "href": "https://www.w3.org/2013/12/ceddl-201312"
```

Comment:
Specref won't let you create two entries for the same URL for now (arguably that's a limitation of the software, but I don't have the resources to fix it in a good enough eay).